Directory.Build.props

```diff
@@ -4,4 +4,13 @@
     <__XFBuildTasksLocation Condition="'$(__XFBuildTasksLocation)' == '' AND '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory).nuspec\netstandard2.0\</__XFBuildTasksLocation>
     <__XFBuildTasksLocation Condition="'$(__XFBuildTasksLocation)' == ''">$(MSBuildThisFileDirectory).nuspec\net46\</__XFBuildTasksLocation>
   </PropertyGroup>
- </Project> 
+  <PropertyGroup>
+    <CI Condition="'$(TF_BUILD)' == 'true'">true</CI>
```

Comment:
it's good practice to set it to `<CI>false</CI>` before checking this, so you can simplify the conditions elsewhere by just using `Condition="$(CI)"` ;)